tests/integration/scripts/create_kind_cluster.sh

```diff
@@ -14,7 +14,7 @@ fi
 
 # create a cluster with the local registry enabled in containerd
 # also add necessary ingress for traefik (for the integration testing)
-cat <<EOF | kind create cluster --name $kind_cluster --image kindest/node:v1.17.11 --config=-
+cat <<EOF | kind create cluster --name $kind_cluster --image kindest/node:v1.20.7 --config=-
```

Comment:
Is there any reason this change is part of this PR? I'm for it, but curious what prompted the upgrade here.

Reply:
That was unintended. I will revert.

Reply:
Do you want to file a separate ticket to get that updated later?